### ISSUE TITLE
REST /api/status always indicate `"IsActiveNode": false`

### ISSUE THREAD

**Issue by solutionguy** [openark/orchestrator#145]
# REST /api/status not concurring with /web/status
## Orchestrator version 1.5.7-1

It looks like a bug, I wanted to find out which Orchestrator node out of two was active, which in my case is node A, and the URI /web/status does reflect that on both nodes. Also, the Available Nodes list doesn't seem to be populating.

However, /api/status returns isActiveNode: false on both nodes, no matter which one is active.

REST API returns a blank current active node, and always returns a false in the isActiveNode for some reason.

Example return values:
Node A

```
{
  "Code": "OK",
  "Message": "Application node is healthy",
  "Details": {
    "Healthy": true,
    "Hostname": "orc-01.lhrx.somecompany.com",
    "Token": "8df743defc2007211b43375d5e2d4351eef4c49d9d3c678b9617530f71e3b356",
    "IsActiveNode": false,
    "ActiveNode": "",
    "Error": null,
    "AvailableNodes": null
  }
}
```
Node B
```
{
  "Code": "OK",
  "Message": "Application node is healthy",
  "Details": {
    "Healthy": true,
    "Hostname": "orc-02.amsx.somecompany.com",
    "Token": "62ce620995be57a5a305c7d94cbfd553491c7e45e2aefbc651234773af55aa21",
    "IsActiveNode": false,
    "ActiveNode": "",
    "Error": null,
    "AvailableNodes": null
  }
}
```

**Comment by shlomi-noach** [comment]
I renamed this Issue.

**Comment by shlomi-noach** [comment]
This was introduced by @grierj via the `StatusSimpleHealth` config, in an attempt to reduce load from load balancer checks.

@solutionguy in the meantime please set `"StatusSimpleHealth": false` in configuration. 

@grierj I'm working to change this behavior and remove this config variable. I suggest a completely different URL, such as:

- `/api/lb-check` (does not run database check)
- `/api/db-check` (runs the `Ping` you implemented)

Thoughts?

**Comment by solutionguy** [comment]
Thanks @shlomi-noach, will test and revert.

**Comment by solutionguy** [comment]
@shlomi-noach I have tested with StatusSimpleHealth turned off, and it does get the status back correctly via /api/status. Thanks

**Comment by grierj** [comment]
@shlomi-noach As long as we continue to allow custom status url endpoint overrides, I have no problem with splitting the status checks.  Square, for instance, has a standard status endpoint that we expect everywhere, so we just need to be able to continue making something respond on it.

**Comment by mcrauwel** [comment]
I encountered the same issue where `/web/status` and `/api/status` did not match. 
Setting the `StatusSimpleHealth` to 'false' solved that issue.

Running version 2.1.4

**Comment by shlomi-noach** [comment]
Thanks -- yes, this hasn't been fixed yet :(

**Comment by choadrocker** [comment]
Setting `StatusSimpleHealth` to false did not solve this for me on 3.0.2 fwiw

**Comment by shlomi-noach** [comment]
Fix to be released in `v3.0.3`, later this week.

**Comment by shlomi-noach** [comment]
Fixed in https://github.com/github/orchestrator/releases/tag/v3.0.3
